### PR TITLE
[blog] Embed TechDocs video

### PR DESCRIPTION
**Pull request by stefanalund**
## Hey, I just made a Pull Request!

Screenshot 2020-09-09 at 08 44 06

#### :heavy_check_mark: Checklist

- [ ] All tests are passing `yarn test`
- [ ] Screenshots attached (for UI changes)
- [ ] Relevant documentation updated
- [ ] Prettier run on changed files
- [ ] Tests added for new functionality
- [ ] Regression tests added for bug fixes

### CODE REVIEW

**Review by freben — APPROVED** [review]
Pity that iframes have those fixed sizes - works poorly for smaller screens

File: microsite/blog/2020-09-08-announcing-tech-docs.md [hunk 1]
```diff
@@ -6,6 +6,9 @@ authorURL: https://github.com/garyniemen
 
 Since we [open sourced Backstage](https://backstage.io/blog/2020/03/16/announcing-backstage), one of the most requested features has been for a technical documentation plugin. Well, good news. The first open source version of TechDocs is here. Now let’s start collaborating and making it better, together.
 
+<iframe width="780" height="440" src="https://www.youtube.com/embed/mOLCgdPw1iA" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen>
+</iframe>
+
```
Comment (emmaindal) [inline microsite/blog/2020-09-08-announcing-tech-docs.md:11]: The video is linked further down in the blog (line 20). Could we embed the video there instead?

Comment (stefanalund) [reply microsite/blog/2020-09-08-announcing-tech-docs.md:11]: We could but then it is show below the preview if you go to /blog

Comment (emmaindal) [reply microsite/blog/2020-09-08-announcing-tech-docs.md:11]: ahh I see, nice, then we maybe can just delete the lines where it's linked out to?

Comment (stefanalund) [reply microsite/blog/2020-09-08-announcing-tech-docs.md:11]: Feel free to update that Emma I’ll merge this one now.

**Pull request merged**